source/views/sis/course-search/row.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import * as React from 'react'
+import {Text, StyleSheet} from 'react-native'
+import type {CourseType} from '../../../lib/course-search/types'
+import {ListRow, Title} from '../../components/list'
+import {Row} from '../../components/layout'
+import {deptNum} from './'
+import moment from 'moment-timezone'
+import {convertTimeStringsToOfferings} from 'sto-sis-time-parser'
+import {formatDayAbbrev} from './lib/format-day'
+import sortBy from 'lodash/sortBy'
+const CENTRAL_TZ = 'America/Winnipeg'
+
+type Props = {
+	course: CourseType,
+	onPress: CourseType => any,
+}
+
+export class CourseRow extends React.PureComponent<Props> {
+	onPress = () => {
+		this.props.onPress(this.props.course)
+	}
+
+	render() {
+		const {course} = this.props
+		const groupings = convertTimeStringsToOfferings(course, {groupBy: 'sis'})
+		const formattedGroupings = groupings.map(grouping => {
+			const times = grouping.times
+				.map(time => {
+					const start = moment
+						.tz(time.start, 'hmm', CENTRAL_TZ)
+						.format('h:mm A')
```

Comment:
If you extract this format pattern into a variable named `fmt`, I think prettier will collapse both these chains onto single lines.